### PR TITLE
feat(*): Map Declaration Docs

### PR DESCRIPTION
**Pull request by jonathan-casey**
### Description
Adds initital documentation for Map type
### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [x] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`

### CODE REVIEW

**Review by DianaLease — APPROVED** [review]
LGTM

File: docs/design/specification/model-maps.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Supported Values
+
+- Primitives: String, DateTime, Integer, Long, Double, Boolean
+- Scalars: String, Integer, Long, Double, Boolean
+- Concepts: Identified & Non-Identified
+
+```
+map <name> {
+    o <required Concerto type>
```
Comment (mttrbrts) [inline docs/design/specification/model-maps.md:21]: Indicate that this is a key value

File: docs/design/specification/model-maps.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+id: model-maps
+title: Maps
+sidebar_position: 9
+---
+A Map is a type which models data as key-value pairs. Conceptually, it relates closely to a Map type in Javascript or C#.
+
+## Supported Keys
```
Comment (mttrbrts) [inline docs/design/specification/model-maps.md:8]: Supported key types

File: docs/design/specification/model-maps.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+id: model-maps
+title: Maps
+sidebar_position: 9
+---
+A Map is a type which models data as key-value pairs. Conceptually, it relates closely to a Map type in Javascript or C#.
+
```
Comment (mttrbrts) [inline docs/design/specification/model-maps.md:7]: Add a comment about how you turn on this feature

**Review by mttrbrts — CHANGES_REQUESTED** [review]

File: docs/design/specification/model-maps.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Example:
+
+```js
+namespace org.acme.hr@1.0.0
+
+map Rolodex  {
+    o String
+    o String
+}
```
Comment (mttrbrts) [inline docs/design/specification/model-maps.md:36]:
```suggestion
map Timeline {
  o DateTime
  o Activity
}
```

File: docs/design/specification/model-maps.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+Types used in a Map may be imported from external namespaces in a manner similar to other [imports](./model-imports.md).
+
+```js
+namespace org.acme.hr@1.0.0 
+import com.my.cool@1.0.0.Thing
+
+map Rolodex  {
+    o String
+    o Thing
+}
```
Comment (mttrbrts) [inline docs/design/specification/model-maps.md:48]:
```suggestion
import com.my.cool@1.0.0.Activity

map Timeline {
  o DateTime
  o Activity
}
```

**Pull request merged**